Dockerfile

```
@@ -1,18 +1,22 @@
 FROM golang:1.11-stretch AS builder
-WORKDIR /go/src/github.com/pusher/oauth2_proxy
+
+# Download tools
+RUN wget -O $GOPATH/bin/dep https://github.com/golang/dep/releases/download/v0.5.0/dep-linux-amd64
+RUN chmod +x $GOPATH/bin/dep
+
+# Copy sources
+WORKDIR $GOPATH/src/github.com/pusher/oauth2_proxy
 COPY . .
 
 # Fetch dependencies
-RUN wget -O dep https://github.com/golang/dep/releases/download/v0.5.0/dep-linux-amd64
-RUN chmod +x dep
-RUN mv dep $GOPATH/bin/dep
 RUN dep ensure --vendor-only
 
-# Build image
-RUN ./configure && make clean oauth2_proxy
+# Build binary
+RUN ./configure && make build.static && strip oauth2_proxy.static
 
-# Copy binary to debian
-FROM debian:stretch
-COPY --from=builder /go/src/github.com/pusher/oauth2_proxy/oauth2_proxy /bin/oauth2_proxy
+# Copy binary to alpine
+FROM alpine:latest
```

I'd rather we didn't use a `latest` tag here as it is considered bad practice, can we use `3.8` instead please
```suggestion
FROM alpine:3.8
```